bin/alluxio-monitor.sh

```diff
@@ -175,27 +175,44 @@ run_monitors() {
     nodes=$(echo -e "${nodes}" | tail -n+2)
     if [[ $? -ne 0 ]]; then
       # if there is an error, print the log tail for the remaining master nodes.
-      for node in $(echo ${nodes}); do
-        run_on_node ${node} "${BIN}/alluxio-monitor.sh" -L "${node_type}"
-      done
+      batch_run_on_nodes "$(echo ${nodes})" "${BIN}/alluxio-monitor.sh" -L ${node_type}
     else
       HA_ENABLED=$(${BIN}/alluxio getConf ${ALLUXIO_MASTER_JAVA_OPTS} alluxio.zookeeper.enabled)
       JOURNAL_TYPE=$(${BIN}/alluxio getConf ${ALLUXIO_MASTER_JAVA_OPTS} alluxio.master.journal.type | awk '{print toupper($0)}')
       if [[ ${JOURNAL_TYPE} == "EMBEDDED" ]]; then
         HA_ENABLED="true"
       fi
       if [[ ${HA_ENABLED} == "true" ]]; then
-        for node in $(echo ${nodes}); do
-          run_on_node ${node} "${BIN}/alluxio-monitor.sh" ${mode} "${node_type}"
-        done
+        batch_run_on_nodes "$(echo ${nodes})" "${BIN}/alluxio-monitor.sh" ${mode} ${node_type}
       fi
     fi
   else
-    for node in $(echo "${nodes}"); do
-      run_on_node ${node} "${BIN}/alluxio-monitor.sh" ${mode} "${node_type}"
-    done
+    batch_run_on_nodes "$(echo ${nodes})" "${BIN}/alluxio-monitor.sh" ${mode} ${node_type}
```

Comment:
should we still quote the `${node_type}` here? same for other locations

Reply:
Test will tell.

Reply:
It works this way, I guess because our node types are always single word.

Reply:
It might not be a bad idea to quote these anyway? I would prefer that anyways, otherwise LGTM

Reply:
how about `${mode}` ?

Reply:
I think generally it is good practice to quote put variables expansions within quotations anyways. I would not be opposed to quoting both

Reply:
Makes sense. Done for both.